openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/targets/EXAMPLE/lib/my_reject_protocol.rb

```diff
@@ -0,0 +1,21 @@
+require 'openc3/interfaces/interface'
+require 'openc3/interfaces/protocols/protocol'
+require 'openc3/api/api'
+
+module OpenC3
+  class MyRejectProtocol < Protocol
+    include Api
+
+    def write_packet(packet)
+      if packet.packet_name == 'START'
```

Comment:
There is no START command ... I assume this is intentional as to not break a bunch of stuff. Probably deserves a comment.

Reply:
START is the only command for the EXAMPLE target.